### PR TITLE
Banner design tool - expandable

### PR DESCRIPTION
**Pull request by tomrf1**
The form is quite long.
This PR uses the Accordion component to make each section expandable
Screenshot 2023-10-11 at 09 31 36

### CODE REVIEW

File: public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx [hunk 1]
```diff
@@ -26,12 +26,39 @@ type Props = {
   onChange: (design: BannerDesign) => void;
 };
 
-export const useLocalStyles = makeStyles(({}: Theme) => ({
+export const useStyles = makeStyles(({ spacing, palette }: Theme) => ({
+  container: {
```
Comment (tomrf1) [inline public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx:30]: some of this copied over from `testEditorStyles.ts`

File: public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx [hunk 2]
```diff
@@ -108,101 +134,122 @@ const BannerDesignForm: React.FC<Props> = ({
 
   return (
     <div className={classes.container}>
-      <div className={classes.sectionContainer}>
-        <Typography variant={'h3'} className={classes.sectionHeader}>
+      <Accordion className={classes.accordion}>
```
Comment (tomrf1) [inline public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx:137]: It's a messy diff, but this essentially just swaps each div/Typography for an Accordion

Comment (tjmw) [reply public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx:137]: 👍 It's a good use case for [hiding whitespace](https://github.com/guardian/support-admin-console/pull/508/files?diff=unified&w=1), the diff definitely gets simpler!

**Review by tjmw — APPROVED** [review]
👍  Great!

**Pull request merged**